packages/commonwealth/client/scripts/controllers/app/webWallets/phantom_web_wallet.ts

```diff
@@ -30,8 +33,25 @@ class PhantomWebWalletController implements IWebWallet<string> {
     return this._accounts || [];
   }
 
-  public async signWithAccount(account: Account): Promise<string> {
-    const encodedMessage = new TextEncoder().encode(account.validationToken);
+  async getChainId() {
+    return app.chain?.id || this.defaultNetwork;
+  }
+
+  public async getRecentBlock(chainIdentifier: string) {
+    // TODO: Use our own Solana RPC - is this configurable?
```

Comment:
Yes -- see `packages/commonwealth/client/scripts/controllers/chain/solana/chain.ts`

Reply:
Ok found it - we can use `app.chain.meta.node.url`

Reply:
Ok, we have decided against implementing block signing for Solana so this will be removed